### PR TITLE
Fixed #422

### LINKED ISSUE #422: Push Notification does to save it setting on exit
After user's turn off push notification from settings, it turns on automatically after user's exit and relaunch the app. 
`It does not keep it setting like the Dark Theme option does`

### PR DESCRIPTION
**Pull request by mistikk**
### Steps to reproduce
Change notification settings and reload application

### Issue number
Fixed #422

### CODE REVIEW

File: src/screens/settings/container/settingsContainer.js [hunk 1]
```diff
@@ -169,7 +169,6 @@ const mapStateToProps = state => ({
   selectedLanguage: state.application.language,
   selectedApi: state.application.api,
   selectedCurrency: state.application.currency,
-  isNotificationOpen: state.application.isNotificationOpen,
   isDarkTheme: state.application.isDarkTheme,
   notificationSettings: state.application.isNotificationOpen,
```
Comment (ue) [inline src/screens/settings/container/settingsContainer.js]:
```suggestion

```

File: src/screens/settings/container/settingsContainer.js [hunk 2]
```diff
@@ -169,7 +169,6 @@ const mapStateToProps = state => ({
   selectedLanguage: state.application.language,
   selectedApi: state.application.api,
   selectedCurrency: state.application.currency,
```
Comment (ue) [inline src/screens/settings/container/settingsContainer.js]:
```suggestion
  selectedCurrency: state.application.currency,
  isNotificationOpen: state.application.isNotificationOpen,
```

File: src/screens/settings/screen/settingsScreen.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ class SettingsScreen extends PureComponent {
       selectedLanguage,
       selectedApi,
       selectedCurrency,
-      isNotificationOpen,
+      notificationSettings,
```
Comment (ue) [inline src/screens/settings/screen/settingsScreen.js]:
```suggestion
      isNotificationOpen,
```

File: src/screens/settings/screen/settingsScreen.js [hunk 4]
```diff
@@ -98,7 +98,7 @@ class SettingsScreen extends PureComponent {
             })}
             type="toggle"
             actionType="notification"
-            isOn={isNotificationOpen}
+            isOn={notificationSettings}
```
Comment (ue) [inline src/screens/settings/screen/settingsScreen.js]:
```suggestion
            isOn={isNotificationOpen}
```

**Comment by mistikk** [comment]
updated @ue

**Pull request merged**